README.md

```diff
@@ -12,6 +12,10 @@ By default, the service runs on port 8990.
 
 ```POST /gvms-haulier-status/movements```
 
+with header :
```

Comment:
minor: rather than repeating this for each endpoint we can have Authorization section to include this info ? but no worries
minor: as you are editing this file, please fix the styling for these bits `* **Code:**` ? dont think they appear as expected in the readme, but no worries we have this issue across other services too :)

Reply:
thanks @sureshhmrc, I've updated as suggested